specs/bep-v1.rst

```diff
@@ -1137,59 +705,12 @@ by the actions of other cluster devices.
 Message Limits
 --------------
 
-An implementation MAY impose reasonable limits on the length of messages
-and message fields to aid robustness in the face of corruption or broken
-implementations. These limits, if imposed, SHOULD NOT be more
-restrictive than the following. An implementation should strive to keep
-messages short and to the point, favouring more and smaller messages
-over fewer and larger. For example, favour a smaller Index message
-followed by one or more Index Update messages rather than sending a very
-large Index message.
-
-=================== =================== =============
-Message Type        Field               Limit
-=================== =================== =============
-**All Messages**
------------------------------------------------------
-|                   Total length        512 MiB
-
-**Index and Index Update Messages**
------------------------------------------------------
-|                   Folder              64 bytes
-|                   Number of Files     1.000.000
-|                   Name                8192 bytes
-|                   Number of Blocks    10.000.000
-|                   Hash                64 bytes
-|                   Number of Counters  1.000.000
-
-**Request Messages**
------------------------------------------------------
-|                   Folder              64 bytes
-|                   Name                8192 bytes
-
-**Response Messages**
------------------------------------------------------
-|                   Data                256 KiB
-
-**Cluster Config Message**
------------------------------------------------------
-|                   Number of Folders   1.000.000
-|                   Number of Devices   1.000.000
-|                   Number of Options   64
-|                   Key                 64 bytes
-|                   Value               1024 bytes
-
-**Download Progress Messages**
------------------------------------------------------
-|                   Folder              64 bytes
-|                   Number of Updates   1.000.000
-|                   Name                8192 bytes
-|                   Number of Indexes   1.000.000
-=================== =================== =============
-
-The currently defined values allow maximum file size of 1220 GiB
-(10.000.000 x 128 KiB). The maximum message size covers an Index message
-for the maximum file.
+An implementation MAY impose reasonable limits on the length of messages and
+message fields to aid robustness in the face of corruption or broken
+implementations. An implementation should strive to keep messages short
+and to the point, favouring more and smaller messages over fewer and larger.
+For example, favour a smaller Index message followed by one or more Index
+Update messages rather than sending a very large Index message.
```

Comment:
Does this mean recipients can reject messages? If it means senders should limit messages sizes, then I think it should say that more clearly. Also, guidelines are good, even if it's just saying what the reference implementation does.